test/jasmine/tests/lib_test.js

```diff
@@ -2631,9 +2631,8 @@ describe('Queue', function() {
             return Plotly.relayout(gd, 'updatemenus[0]', null);
         })
         .then(function() {
-            // buttons have been stripped out because it's an empty container array...
```

Comment:
turns out this was a mistake, applying our logic self-consistently we should *not* delete this array.